jobseeker/templates/jobseeker/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<head>
+<title>Job Seeker</title>
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+</head>
+
+<body>
+<!-- Header -->
+<header class="w3-container w3-red w3-center" style="padding:128px 16px">
+  <h1 class="w3-margin w3-jumbo">JOB SEEKER</h1>
+  <p class="w3-xlarge">Mentors: Yariv Rachmani, Kasem Alem, Ronnie Rasouli<br>Students: Omer Pahima, Amit Cohen, Shirley Fichman, Dean Biton, Tal Jacob</p>
+</header>
+<!-- Grid -->
+<div class="w3-row-padding w3-padding-64 w3-container">
+  <div class="w3-content">
+    <div class="w3-twothird">
+      <h1>Project Description</h1>
+      <h5 class="w3-padding-32">Job Seeker is a job board website that is used both by job-seekers and employers:<br>Job-seekers can use Job Seeker to search for new job opportunities in their area and profession.<br>Employers can use Job Seeker to advertise their open positions and find new candidates.</h5>
+    </div>
+    <div class="w3-third w3-center">
```

Comment:
Tried to add the logo here but it failed for some reason..

Reply:
Your'e gonna need to make a few changes in order to load the logo file:

1. add the following img tag inside the div at line 20 (if that where you wish to place it) `<img src="{% static 'jobseeker/logo.png' %}" />`
2. Add the following line to the top of this HTML page: `{% load static %}`
3. Add the following directories to the jobseeker app directory: `static/jobseeker/` and place the logo file there.
4. Add the following urls.py file in the jobseeker app directory:
  ```
from django.urls import path
  from . import views
  
  urlpatterns = [
      path('', views.index, name='index'),
  ]
```
5. Change the app urls.py file to look like the following:
  ```
from django.contrib import admin
  from django.urls import path, include
  
  urlpatterns = [
      path('admin/', admin.site.urls),
      path('', include('jobseeker.urls')),
  ]
```

Reply:
Works! thanks a lot